dcat/index.html

```diff
@@ -3323,30 +3328,264 @@ <h2>Version types</h2>
+<p>The other sections include only editorial changes.</p>
+</aside>
+
+<p>The notion of <em>version</em> is often used as a generic term to denote some kind of relationship between a resource and a derived one. Examples, among others, include revisions, editions, adaptations, and translations.</p>
+
+<p>This section focuses specifically on how to use DCAT to describe versions resulting from a revision - i.e., from changes occurring to a resource as part of its life-cycle.</p>
+
+<!--
+<p>Henceforth, the words <em>version</em> and <em>versioning</em> must be intended in this restrictive sense.</p>
+-->
+
+<p>For this purpose, DCAT builds upon existing vocabularies, in particular the versioning component of the [[?PAV]] ontology, and the relevant terms from [[?DCTERMS]], [[?OWL2-OVERVIEW]], and [[?VOCAB-ADMS]].</p>
+
+<p>It is important to note that versioning can be applied to any of the first class citizens DCAT resources, including Catalogs, Catalog Records, Datasets, Distributions.</p>
+
+<p>Note also that the DCAT approach described in the following sections is meant to be complementary with those already used in specific types of resources (e.g., [[OWL2-OVERVIEW]] provides a set of versioning properties for ontologies), as well as in given domains and communities.</p> 
+
+<aside class="note">
+<p>The notion of version is very much related to the community practices, the data management policy and the workflows in place. It is up to data providers to decide when and why a new version should be released. For this reason, DCAT refrains from providing definitions or rules about when changes in a resource should turn into a new release of it, and refers for guidance to [[?DWBP]] (<a data-cite="?DWBP#dataVersioning">&sect;&nbsp;<span class="secno">8.6 </span>Data Versioning</a>).</p>
+</aside>
+
+<section id="version-relationships">
+
+<h2>Relationships between versions</h2>
+
+<p>DCAT supports three kinds of relationships between versions:</p>
+<ol>
+<li>Those indicating the version chain and hierarchy (the version history).</li>
+<li>Those indicating whether a version is replaced/superseded by another one.</li>
+<li>Those indicating whether a version is backward compatible with another one.</li>
+</ol>
+
+<section id="version-history">
+
+<h2>Version chains and hierarchies</h2>
+
+<p>DCAT defines specific properties for describing version history, aligned with the corresponding [[?PAV]] ones:</p>
+
+<ul>
+<li><code>dcat:previousVersion</code> (equivalent to <a data-cite="?PAV#d4e459"><code>pav:previousVersion</code></a>)</li>
+<li><p><code>dcat:hasVersion</code> (equivalent to <a data-cite="?PAV#d4e395"><code>pav:hasVersion</code></a>), plus the following additional properties:</p>
+<ul>
+<li><code>dcat:isVersionOf</code> (inverse of <code>dcat:hasVersion</code>);</li>
```

Comment:
In general inverse-properties are to be avoided (you can always add the reverse triple to a graph) but occasionally there is a pretty compelling case from usage. This is probably such.

Reply:
I agree. In the text I tried to make it clear that the only key properties are dcat:hasVersion and dcat:previousVersion, whereas the other ones should be used if needed by the relevant use case.